minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
> Mini runner is user facing

It is, but I don't see a place where we actually leak MiniAccumuloCluster/Config through that class. Did I miss something? If we want to be really picky (and these changes go on the impl), we should not even expose the new property via MiniAccumuloRunner. Is this what you were pointing to?